android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUITabbedBar.java

```diff
@@ -133,9 +134,14 @@ protected void onLayout(boolean changed, int l, int t, int r, int b)
 				// no selected color specified but a text color -> use that
 				selectedTextColor = textColor;
 			}
-
 			this.tabLayout.setTabTextColors(textColor, selectedTextColor);
 		}
+		if (this.tabLayout.getTabAt(0).getIcon() != null
```

Comment:
Could this crash if no tab is set at all (e.g. `this.tabLayout.getTabAt(0)` being `null`)?